### DISCUSSION TITLE
server/api code cannot import @nuxt/kit

### DISCUSSION THREAD

**Discussion by TokyoPiglet** [nuxt/nuxt#25557] [Questions] [answered]
My code at /server/api/read/[file].ts stopped compiling after upgrading Nuxt from 3.8 to 3.10. 

```
import fs from 'node:fs';
import { sendStream } from 'h3'
import { defineNuxtModule, resolvePath } from '@nuxt/kit'
import { join } from 'pathe'

export default defineEventHandler(async (event) => {
  const file = event.context.params.file;
  const entry = await resolvePath('public')
  return sendStream(event, fs.createReadStream(join(entry, file)))
}
```
The error is:
```
 ERROR  This module cannot be imported in server runtime. [importing @nuxt/kit from server/api/read/[file].ts] 
```
How can I get the path to the application's public directory on the server's local storage without using resolvePath()?

**Comment by arashsheyda** [comment, marked as answer]
@TokyoPiglet you can use [`pathe`](https://unjs.io/packages/pathe)

**Reply by TokyoPiglet** [reply]
Thank you. I deleted the import from '@nuxt/kit', added 'resolve' to the import from 'pathe', and changed the usage to: await resolve('public'). It works.
